bin/main.rs

```diff
@@ -333,12 +343,12 @@ async fn main() -> anyhow::Result<()> {
 
 #[derive(Clone)]
 struct CommandWorkerCreator {
-    pool: ControlClientConstructor,
+    pool: ControlClientConstructor<DirectKvStore>,
 }
 
 #[async_trait::async_trait]
 impl WorkerCreator for CommandWorkerCreator {
-    type Output = CommandWorker;
+    type Output = CommandWorker<DirectKvStore>;
```

Comment:
Is this intended to be directly querying or does it just not matter since for commands we're just publishing to the KV bucket?